tools/icu/icutrim.py

```diff
@@ -317,12 +317,12 @@ def removeList(count=0):
             erritems = fi.readlines()
             fi.close()
             #Item zone/zh_Hant_TW.res depends on missing item zone/zh_Hant.res
-            pat = re.compile("""^Item ([^ ]+) depends on missing item ([^ ]+).*""")
+            pat = re.compile(rb"^Item ([^ ]+) depends on missing item ([^ ]+).*")
```

Comment:
This leads to a syntax error with Python 2.7, which is still supported for building Node.js (https://github.com/nodejs/node/blob/10d378e947565d4aa786f9f5bd69f55c208faf0f/BUILDING.md#note-about-python-2-and-python-3).

```suggestion
            pat = re.compile(bytes(r"^Item ([^ ]+) depends on missing item ([^ ]+).*"))
```

Should fix that, I think, but I’m also getting other errors when using Python 3 when running this file:

```
Traceback (most recent call last):
  File "icutrim.py", line 163, in <module>
    config=json.load(fi)
  File "/usr/lib/python3.5/json/__init__.py", line 268, in load
    parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
  File "/usr/lib/python3.5/json/__init__.py", line 312, in loads
    s.__class__.__name__))
TypeError: the JSON object must be str, not 'bytes'
```

(which, interestingly, this PR did not touch?)

Reply:
thanks for your help @addaleax I updated this part of code.
I hopenow that will works  for you :-)